versions/3.0.md

```diff
@@ -1102,10 +1102,10 @@ application/json:
   schema:
     $ref: "#/definitions/Pet"
   examples:
+    # converted directly from YAML to JSON
     - name: Fluffy
       petType: Cat
-    - name: Rover
-      petType: Frog
+    - {"name": "Rover", "petType": "Frog"}
```

Comment:
I find the json encoding inside the YAML (while correct) to be confusing; I don't think it is done elsewhere in the 3.0 spec.